app/controllers/short_url_requests_controller.rb

```diff
@@ -49,6 +51,21 @@ def reject
     redirect_to short_url_requests_path
   end
 
+  def destroy
+    router.delete_route("#{@short_url_request.from_path}")
```

Comment:
Oh. I copied from a previous example of a data migration I did to change a slug in Whitehall while on second-line which needed a route deleting. Probably not the best place to copy from!

I'll try to understand how the publishing-api and "gone" works... :-\